lib/rectangles.js

```diff
@@ -0,0 +1,27 @@
+var rectangle = {width: 10, height: 20};
+
+function area(rectangle) {
+  return (rectangle["width"] * rectangle["height"]);
```

Comment:
Is there a reason you're using bracket vs. dot notation to access the width and height here?